pyproject.toml

```diff
@@ -41,13 +41,12 @@ classifiers = [
 ################################################################################################
 dependencies = [
     # License: Apache 2.0
-    # transitive dependencies:
-    #   urllib3: MIT
-    #   aiohttp: Apache 2.0
-
     "elasticsearch[async]==8.6.1",
     "elastic-transport==8.4.1",
+    # License: MIT
     "urllib3==1.26.19",
+    # License: Apache 2.0
+    "aiohttp==3.10.8",
```

Comment:
Nit: The latest 3.10.x is 3.10.11, actually.

Reply:
I think we were inspired by the state of local env too much, so https://github.com/elastic/rally/pull/1890/commits/eddc70cf3c9ea6f50d1d61243d2b81b8f3961548.